TERMSOFSERVICE.md

```diff
@@ -420,64 +457,68 @@ reasonable use, we have the right to block access to Tideways until such
 default has been rectified. In such cases we will normally notify you of such
 blocking. Our other legal and contractual rights shall remain unaffected.
 
-# 17. Change in ToS
+# 17. Changes to ToS
 
-17.1. We reserve the right to change the ToS at any time with effect from a
```

Comment:
We reworded this paragraph from "We" to "Tideways". The meaning of this paragraphs have not changed.